### PR TITLE
add system install powershell

### LINKED ISSUE #1067: Add Powershell as system install
## Expected Behaviour

Add [powershell](https://github.com/PowerShell/PowerShell/releases/) as system install in arkade

### PR DESCRIPTION
**Pull request by nitishkumar71**
## Description

Add [powershell](https://github.com/PowerShell/PowerShell/releases/) as system install application
## Motivation and Context

- [x] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))
Closes #1067 

## How Has This Been Tested?

Using command `sudo ./arkade system install pwsh` produced below output
```
Installing Powershell to /usr/local/bin
Installing version: v7.4.2 for: x64
Downloading from: https://github.com/PowerShell/PowerShell/releases/download/v7.4.2/powershell-7.4.2-linux-x64.tar.gz
67.69 MiB / 67.69 MiB [--------------------------------------------------------------------------------------------------------------------------------------------------------------] 100.00%
Downloaded to: /tmp/arkade-3499737583/powershell-7.4.2-linux-x64.tar.gz
Unpacking Powershell to: /usr/local/bin/powershell
```

If updating or adding a new CLI to `arkade get`, run:

```
go build && ./hack/test-tool.sh TOOL_NAME
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [ ] I have updated the list of tools in README.md if (required) with `./arkade get --format markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

File: cmd/system/powershell.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+		Use:   "pwsh",
+		Short: "Install Powershell",
+		Long:  `Install Powershell cross-platform task automation solution.`,
+		Example: `  arkade system install pwsh
+  arkade system install pwsh --version <version>`,
+		SilenceUsage: true,
+	}
+
+	command.Flags().StringP("version", "v", "", "The version or leave blank to determine the latest available version")
+	command.Flags().String("path", "/usr/local/bin", "Installation path, where a powershell subfolder will be created")
```
Comment (alexellis) [inline cmd/system/powershell.go:29]: Hi, I know most of what we install goes here, but this isn't the recommended path for PowerShell

Could you consult the docs and update?

Comment (nitishkumar71) [reply cmd/system/powershell.go]: Done

**Comment by nitishkumar71** [comment]
Retested the command after changes and it produced below output
```
sudo ./arkade system install pwsh
Installing version: v7.4.2 for: x64
Installing Powershell to /opt/microsoft/powershell/7
Downloading from: https://github.com/PowerShell/PowerShell/releases/download/v7.4.2/powershell-7.4.2-linux-x64.tar.gz
67.69 MiB / 67.69 MiB [--------------------------------------------------------------------------------------------------------------------------------------------------------------] 100.00%
Downloaded to: /tmp/arkade-3612803353/powershell-7.4.2-linux-x64.tar.gz
Unpacking Powershell to: /opt/microsoft/powershell/7/powershell
Creating Symbolic link to: /usr/bin/pwsh
```

File: cmd/system/powershell.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+				return err
+			}
+			version = v
+		} else if !strings.HasPrefix(version, "v") {
+			version = "v" + version
+		}
+
+		fmt.Printf("Installing version: %s for: %s\n", version, dlArch)
+
+		semVer := strings.TrimPrefix(version, "v")
```
Comment (alexellis) [inline cmd/system/powershell.go:72]: This package may cover edge cases better: https://github.com/Masterminds/semver

Comment (nitishkumar71) [reply cmd/system/powershell.go]: Done

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**